src/styles/blocks/button.scss

```diff
@@ -0,0 +1,23 @@
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--main-color);
+  color: #fff;
```

Comment:
You should use a CSS variable for the color #fff, to keep the code consistent with other parts of the code using variables.